scalameta/semanticdb/shared/src/main/scala/scala/meta/semanticdb/Symbol.scala

```diff
@@ -1,203 +0,0 @@
-package scala.meta.semanticdb
-
-import scala.compat.Platform.EOL
-import scala.meta.internal.semanticdb3.Scala._
-import scala.meta.internal.semanticdb3.Scala.{Descriptor => d}
-
-sealed trait Symbol extends Product {
-  def syntax: String
-  def structure: String
-}
-
-object Symbol {
-  case object None extends Symbol {
-    override def toString = syntax
-    override def syntax = Symbols.None
-    override def structure = s"""Symbol.None"""
-  }
-
-  final case class Local(id: String) extends Symbol {
-    override def toString = syntax
-    override def syntax = Symbols.Local(id)
```

Comment:
Now that this line is gone, I suggest we delete `def Local(id: String): String = ???` from the public API.